src/core/BufferGeometry.js

```diff
@@ -358,6 +374,13 @@ THREE.BufferGeometry.prototype = {
 
 			var attribute = this.attributes.position;
 
+			if(attribute !== undefined && attribute.array instanceof DataView)
+			{
+
+				console.error("THREE.BufferGeometry.updateFromObject: Cannot modify geometry when BufferAttribute.array is a DataView");
+				return;
+
+			}
```

Comment:
DRY, please!

I'd expect
```javascript
this.checkNoDataView( "'updateFromObject" );
```

All the strings end up redundantly in the library source and bloat it up...